app/models/user.rb

```diff
@@ -4,7 +4,8 @@ class User < ApplicationRecord
           :recoverable, :rememberable, :trackable, :validatable,
           :confirmable, :omniauthable
   include DeviseTokenAuth::Concerns::User
-  has_many :articles, through: :authorships
+  has_many :authorships
+  has_many :articles, through: :authorships, dependent: :destroy
   has_many :comments
```

Comment:
Do we want to preserve comments? If so we need to make sure it isn't an error to have a comment with no author